sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_extensions.cs

```diff
@@ -0,0 +1,367 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Messaging.WebPubSub
+{
+    /// <summary>
+    /// Azure Web PubSub Service Client.
+    /// </summary>
+    [CodeGenSuppress("WebPubSubServiceClient", typeof(string), typeof(Uri), typeof(WebPubSubServiceClientOptions))]
+    public partial class WebPubSubServiceClient
+    {
+        private AzureKeyCredential credential;
+
+        private const string JsonContent = "application/json";
+
+        /// <summary>
+        /// The hub.
+        /// </summary>
+        public string Hub => hub;
```

Comment:
Nit - this property and `Endpoint` should be `virtual` so they're easier to mock.